CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.0.0-alpha1
+ * New schema backend, which is faster, simpler, and more declarative, enabling more applications and simplifying tooling.  Users of built-in schema types should experience very little or no breakage, but tooling or custom schema types will need to be updated.
```

Comment:
:sheep: might want to mention that we now experimentally support test-check style generation from schema as a concrete example of an application that's enabled